generators/CoreCLRStatefulService/templates/service/app/appPackage/servicePackage/Code/entryPoint.sh

```diff
@@ -9,5 +9,7 @@ check_errs()
 }
 
 DIR=`dirname $0`
+echo 0x3f > /proc/self/coredump_filter
+source ../../../../dotnet-install.sh
```

Comment:
Where is this script placed when template is created?

Reply:
In the main folder. Where the install build and uninstall scripts are located

Reply:
ok..